demo/index.html

```diff
@@ -2,56 +2,46 @@
 <html lang="en">
 <!-- This file is intended to be built using the instructions here: http://highlightjs.readthedocs.io/en/latest/building-testing.html#building -->
 <head>
-  <meta charset="utf-8">
+  <meta charset="utf-8" />
```

Comment:
Why? HTML5 does not need to be valid XHTML nor is there a good reason to force it to be off the top of my head.  Is there a big reason for these changes?

Reply:
No, mostly personal preference.

Reply:
I like this [answer from StackOverflow](https://stackoverflow.com/questions/3558119/are-non-void-self-closing-tags-valid-in-html5):

> **In HTML5, the meaning of <foo /> depends on the type of element.**

> On HTML elements that are designated as void elements (essentially "An element that existed before HTML5 and which was forbidden to have any content"), end tags are simply forbidden. The slash at the end of the start tag is allowed, but has no meaning. It is just syntactic sugar for people (and syntax highlighters) that are addicted to XML.

Since we're using HTML5 not XHTML I think we should be clear and just write clean and simple HTML5, not XHTML.  If we really wanted to write XHTML we should change our DTD to say so, but I don't see any benefit to this.

Reply:
Removed.